src/icons/AccountCircle.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import colors, { Colors } from 'src/styles/colors'
+
+interface Props {
+  size?: number
+  color?: colors
+}
+
+const AccountCircle = ({ size = 24, color = Colors.successDark }: Props) => (
+  <Svg width={size} height={size} fill="none">
```

Comment:
can you include a viewBox param in all the icons? Otherwise adjusting with size won't work as expected

Reply:
Added in 370ffa7e401ae7c2b9dc33be98054855d04e1c6f!